utils/docker/Dockerfile.centos.7

```diff
@@ -194,6 +196,9 @@ RUN if [ "x$BULLSEYE" != "x" ]; then \
       rm -rf bullseye; \
     fi
 
+# Add packages provided by PR-repos that have just been set up.
+RUN dnf -y install python3-pyelftools
```

Comment:
This will need to be moved to the previous block/list (where you added `meson` and `ninja-build` once the PR for `pyelftools` lands.